### ISSUE TITLE
Questions about the design choices for scale_head range and scale_transform in GaussTR

### ISSUE THREAD

**Issue by Yandong-Luo** [hustvl/GaussTR#37]
`scale_head=dict(
            type='MLP',
            input_dim=embed_dims,
            output_dim=3,
            mode='sigmoid',
            range=(1, 16)),`
`@MODELS.register_module()
class MLP(nn.Module):

    def __init__(self,
                 input_dim,
                 hidden_dim=None,
                 output_dim=None,
                 num_layers=2,
                 activation='relu',
                 mode=None,
                 range=None):
        super().__init__()
        hidden_dim = hidden_dim or input_dim * 4
        output_dim = output_dim or input_dim
        self.num_layers = num_layers
        h = [hidden_dim] * (num_layers - 1)
        self.layers = nn.ModuleList(
            nn.Linear(n, k) for n, k in zip([input_dim] + h, h + [output_dim]))
        self.activation = activation
        self.range = range
        self.mode = mode

    def forward(self, x):
        for i, layer in enumerate(self.layers):
            x = getattr(F, self.activation)(
                layer(x)) if i < self.num_layers - 1 else layer(x)

        if self.mode is not None:
            if self.mode == 'sigmoid':
                x = F.sigmoid(x)
            if self.range is not None:
                x = self.range[0] + (self.range[1] - self.range[0]) * x
        return x`
`scales = self.scale_head(x) * self.scale_transform(
            sample_depth, cam2img[..., 0, 0]).clamp(1e-6)`

Hi, thank you for the great work on GaussTR!

While reading the implementation, I had a few questions regarding the design choices for the predicted scale of the 3D Gaussians:

1. Range of [1, 16] in scale_head
In the MLP used for predicting the per-query scale, the output is passed through a sigmoid and then linearly mapped to the range [1, 16].
Could you share the motivation behind choosing this specific range? Was it empirically determined, or based on a particular geometric or physical prior?

2. scale_transform(depth, focal) design
In the scale_transform function:
`def scale_transform(self, depth, focal, multiplier=7.5):
    return depth * multiplier / focal
- What is the reasoning behind this specific form (depth × multiplier ÷ focal)?

- Is there a geometric intuition or empirical observation behind the choice of multiplier = 7.5?`

3. Have you considered directly predicting real-world scales?
Instead of using a relative scale [1, 16] followed by the transform depth * multiplier / focal, was directly regressing physical sizes (e.g., in meters) ever tried? If so, were there any stability or convergence issues?

Looking forward to your insights!
Thanks

**Comment by npurson** [comment]
1. Regarding the ranges for `scale_head` and `multiplier`: These were determined empirically. We visualized a set of randomly initialized Gaussians within the scene to identify a suitable range.
2. Regarding the `scale × depth / focal` formulation: This expression is derived from the principles of perspective projection. The hypothesis is that it simplifies the learning task by normalizing the target scale to the image plane. However, a key limitation is its reliance on the predicted depth. We believe that directly regressing the 3D scale, thereby removing this dependency, is a promising direction for future work.

Hope this helps!

**Comment by Yandong-Luo** [comment]
Thanks for your reply. I'm really appreciate
